src/core/ext/transport/binder/utils/transport_stream_receiver_impl.cc

```diff
@@ -117,7 +117,7 @@ void TransportStreamReceiverImpl::RegisterRecvTrailingMetadata(
 void TransportStreamReceiverImpl::NotifyRecvInitialMetadata(
     StreamIdentifier id, absl::StatusOr<Metadata> initial_metadata) {
   gpr_log(GPR_INFO, "%s id = %d is_client = %d", __func__, id, is_client_);
-  if (!is_client_ && accept_stream_callback_) {
+  if (!is_client_ && accept_stream_callback_ && initial_metadata.ok()) {
```

Comment:
Why this change in this PR?

Reply:
We probably don't want to open a new stream for invalid initial metadata. This was inspired by the server fuzzer and since this is so small I included it here. I can also open a separate PR for this.